frontend/src/app/modules/user/components/user-profile-settings/user-profile-settings.component.ts

```diff
@@ -14,6 +14,9 @@ import { UserService } from '@core/services/user.service';
 export class UserProfileSettingsComponent extends BaseComponent implements OnInit {
     public user: User;
 
+    //fake
+    public userId: number = 3;
```

Comment:
It is easy to forget about this hardcoded value

Reply:
I made id 1 after PR, I implement in a new branch for an authorized user

Reply:
Please do it in this pr